src/components/Tooltip/Tooltip.tsx

```diff
@@ -38,8 +38,8 @@ const Tooltip: React.FC<TooltipProps> = ({
 	auto = true,
 	placement = 'top-center',
 	triggerOffset = 8,
-	delayEnter = 100,
-	delayLeave = 200,
+	delayEnter = 0,
+	delayLeave = 0,
```

Comment:
Not sure why we updated the default values for tooltip as this will modify the existing behavior in other areas of app.

Reply:
It was from feature feedback

Reply:
I will leave this as it is as I'm not sure what is decided here. Will add to my tech debt notes as something to possibly look back at